server/controllers/items/lib/delete_user_items.coffee

```diff
@@ -6,16 +6,5 @@ items_ = __.require 'controllers', 'items/lib/items'
 promises_ = __.require 'lib', 'promises'
 
 module.exports = (userId)->
-  # get user items ids
   items_.byOwner userId
-  .then deleteItemsAndComments.bind(null, userId)
```

Comment:
what about delete comments too?

Reply:
not sure of what you mean

Reply:
garbage collect orphan comments

Reply:
there are no more items comments in the app, trying to delete those was thus legacy code that this PR removes

Reply:
ok